bika/lims/browser/clientfolder.py

```diff
@@ -132,6 +132,9 @@ def __init__(self, context, request):
     def before_render(self):
         """Before template render hook
         """
+        # Call `before_render` from the base class
+        BikaListingView.before_render(self)
```

Comment:
Would you mind to use `super` here instead of calling the named base class directly?
E.g. like here: https://github.com/senaite/senaite.core/blob/master/bika/lims/browser/worksheet/views/folder.py#L205